testutil/compare.go

```diff
@@ -294,3 +299,95 @@ func compareUUID(l, r *Ydb.Value) int {
 		return 0
 	}
 }
+
+// CompareCreateTableOptions compares table options used for creating YDB table.
+//
+// This function should be used only for testing purpose.
+//
+// # Experimental
+//
+// Notice: This API is EXPERIMENTAL and may be changed or removed in a later release.
+func CompareCreateTableOptions(expected, actual []options.CreateTableOption) func(*testing.T) {
```

Comment:
I suggest return result: true/false + diff string.

1. What if I want check about options are not equal?
2. What if I want use it without T (for example internal tests with many times use testing.TB instead of *T), may be other test frameworks with use own types.

Reply:
Decided to use something like

```go
type CreateTopicOptionsDesc struct {
	opts *rawtopic.CreateTopicRequest
}

func ToCreateTopicOptionsDesc(opts ...topicoptions.CreateOption) *CreateTopicOptionsDesc {
	req := &rawtopic.CreateTopicRequest{}
	for _, v := range opts {
		v(req)
	}

	return &CreateTopicOptionsDesc{
		opts: req,
	}
}
```